.github/workflows/mods.yml

```diff
@@ -0,0 +1,21 @@
+on: push
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.CZIBUILDBOT_GITHUB_TOKEN }}
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.2'
+      - name: fix go modules
+        run: |
+          go mod tidy
+          git add go.*
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
not sure about this but, would this work in the "general" case?

Reply:
What do you mean by the general case? Do you mean when there is nothing to do?

Reply:
nvm, I got confused